stable/jenkins/templates/config.yaml

```diff
@@ -155,6 +155,20 @@ data:
       <pendingClasspathEntries/>
     </scriptApproval>
 {{- end }}
+  jenkins.model.JenkinsLocationConfiguration.xml: |-
+    <?xml version='1.1' encoding='UTF-8'?>
+    <jenkins.model.JenkinsLocationConfiguration>
+      <adminAddress>{{ default "nobody@nowhere.io" .Values.Master.JenkinsAdminEmail }}</adminAddress>
```

Comment:
Please document `JenkinsAdminEmail` in the README.  Also, could we just default to an empty string?  Not sure who owns nowhere.io...